resource/subfunctions/signed_program.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Subfunctions
+---
+
+# Signed Program
+
+Resource Administration functions for Signed Program Administration
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _This may not be the only step to manage a **Signed Program** in your environment._
+> _You may also have to refresh the **PROGRAM** class to enact these changes._
+> _Please consult RACF documentation and manuals for an understanding of the **PROGRAM** class._
+
+## `ResourceAdmin.add_signed_program()`
+
+```python
+  def add_signed_program( self, signed_program_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def add_signed_program(self, signed_program_name: str, traits: dict = {}) -> Union[dict, bytes]:
```

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.